applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -355,6 +355,12 @@ def _add_water_variables(self):
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.NORMAL_FLUID_FLUX)
         # Add variables for the water conditions
         self.main_model_part.AddNodalSolutionStepVariable(GeoMechanicsApplication.HYDRAULIC_DISCHARGE)
+        
```

Comment:
I can imagine other integration point results that are interesting for the DGeo packages and therefor could be added:
DEGREE_OF_SATURATION is one of them, but they don't know they want that yet.
PERMEABILITY output could be desirable too.